### ISSUE TITLE
Proper support of data-large attributes

### ISSUE THREAD

**Issue by pdawyndt** [dodona-edu/dodona#3263]
The new lightbox does not properly support `data-large` attributes for `img`-tags. The idea of that attribute is to provide a larger version of an image that is show in the lightbox, instead of the original image used in the description. Now both the original image and the larger version are used by the lightbox (2 images instead of one).

Example [exercise](https://dodona.ugent.be/nl/activities/772656454/)

NOTE: in this example the double display of the first image with a larger version overwrites the second image in the list (a GIF), such that the GIF is not shown in a lightbox; if GIFs are never included in the lightbox list, then this is another unrelated issue

**Comment by jorg-vr** [comment]
I have found this issue to be hard to reproduce.
In firefox everything worked perfectly.
In chrome I have seen the double images once. I think this was because I opened the lightbox before it was fully finished computing. It recovered form this state pretty fast (Removing one of the images and thus only showing a single image as intended)

I think this is a not very important lightbox issue that is hard to encounter in reallity. I would mark this as won't fix

**Comment by chvp** [comment]
I can reproduce this with the exercise mentioned in the original issue, but both `data-large` attributes link to the same image, so I think it is expected behaviour in that case?

**Comment by jorg-vr** [comment]
That is not the actual issue @chvp. This indeed makes it an odd example, but as described in the text this is intended (To avoid the giff)

What is not intented (And as I have mentioned, I myself have seen only once) Is the image being displayed twice in the lightbox side by side (So not in different steps)

It was gone to fast for me to produce a screenshot

**Comment by pdawyndt** [comment]
I have tried to reproduce the issue myself (Chrome) with multiple exercises that use the `data-large` attribute, and I have no longer seen the observed the original issue again. Even the issue with displaying GIFs in the lightbox has gone, so I removed the `data-large` attribute in the exercise used as an example. I want to reserve use of the `data-large` attribute for initially loading a smaller version of large images in the description of an exercise, to speed up loading.

I believe this issue might be closed. Will open it again in case I would bump into the issue again.
